ArduCopter/crash_check.cpp

```diff
@@ -353,8 +329,8 @@ void Copter::parachute_manual_release()
     }
 
     // do not release if vehicle is landed
-    // do not release if we are landed or below the minimum altitude above home
-    if (ap.land_complete) {
+    // land flag cannot be trusted if in standby mode, use armed status instead
+    if ((ap.land_complete && !standby_active) || (standby_active && !motors->armed())) {
```

Comment:
this maybe brings an interesting behavior : 
if a cube in standby crash and reboot , then it will be disarmed and can trigger the parachute. As this is for manual release, that isn't important.